src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import platform as _platform
```

Comment:
Must be given a _ prefix due to name collision in FMUdataioSystemInformationPlatform